src/Page/Story/View.elm

```diff
@@ -23,8 +24,13 @@ view language story =
         call =
             Maybe.withDefault (t HomeCallForStoryP) story.customCall
     in
-    div [ css [ centerContent ] ]
+    div [ css [ storyContainerStyle ] ]
         [ primaryHeader [] story.title
+        , viewBreadcrumbs
+            [ { text = "Home", href = Route.toString Index }
+            , { text = "Stories", href = Route.toString Guides }
```

Comment:
We'll rework this when the search is finished. #309 but I'll include a screenshot for @undividual design review too.